node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/ApplicationMaintainer.java

```diff
@@ -41,7 +43,6 @@ protected final void maintain() {
         for (ApplicationId application : applications) {
```

Comment:
Do we have logging outside of this loop or should we add it to see how long it takes ?

Reply:
Since the deployments will just be queued up I don't think it will help us much, we would need to somehow wait until all submitted jobs were finished then

Reply:
Is it not possible to to do an executor.sync or something to test that it is done.